index.html

```diff
@@ -1644,6 +1644,141 @@ <h4>Background</h4>
+		labor practices in local agriculture. Samira wants to support the efforts of this group and get
+		involved, but she worries that publicly allying with the farm workers will negatively affect her
+		ability to act as a neutral party in labor negotiations. Using DIDs to create a unique identifier
+		for herself in the group at very little cost, she can engage in multi-party secure communication
+		with all of the members of Ahmed’s organization. Messages are secured via mutual authentication
+		between Samira and each of the other members. Using this connection, Samira is able to have private
+		and confidential communications with the other members at any time without fear of censorship or
+		surveillance. Members of the organization are able to have confidence that they’re talking to
+		Samira and she can verify who she is communicating with. Samira can use the same identifier and
+		connection with the group to share credentials that prove she’s a humanitarian worker and attest
+		to her history.
+        </p>
+        <p>
+          Eventually, Samira loses her phone. To make sure no one impersonates her, she rotates the keys on all
+		of the DIDs she is using for communication. Rather than losing access to all of her existing
+		connections, Samira is able to update her service endpoints and routing capabilities to make
+		sure she is still receiving messages securely on the devices she still owns. When she gets a new
+		phone, she can add new keys to her DID Document to continue exchanging secure messages with her
+		connections uninterrupted. Shold Samira want to change the DIDs she’s using, for instance because
+		she’s changing DID method providers, she can send a message to her connections notifying them
+		that she’s deprecating her old DID and using a new DID with authentic digital signatures.
+		Samira is able to manage all of her secure communications across her devices, platforms
+		and applications in a consistent way, thanks to the security features provided by DIDs.
+        </p>
+      </section>
+
+      <section id="messagingChallenges">
+        <h4>Challenges</h4>
+        <p>
+          Samira’s use of a DID-based communication system comes with a number of challenges related to user
+		experience. This includes management of different communication contexts and context-switching, as
+		well as the automation of connection management with things like key rotation, updating service
+		endpoints, and routing via Agents. Users should not be exposed to the details of such coordination,
+		but should be able to interact with, modify, and audit the process as needed. It’s likely that
+		users will be interacting with secure messaging protocols via a number of different interfaces &amp;
+		devices and for a variety of different purposes, though the logic and experience should be
+		coherent and consistent across different platforms.
```

Comment:
```suggestion
          Samira's use of a DID-based communication system comes with a number of user experience challenges.
		These include the management of her different communication contexts and switching between them, as
		well as the automation of connection management with things like key rotation, updating service
		endpoints, and routing via Agents. Users should not be exposed to the details of such coordination,
		but should be able to interact with, modify, and audit the process as needed. It’s likely that
		users will be interacting with secure messaging protocols via a number of different interfaces &amp;
		devices and for a variety of different purposes, though the logic and experience should be
		coherent and consistent across different platforms.
```

Reply:
@philarcher -- This suggestion appears to have been "resolved" without incorporation, nor comment about why it wasn't incorporated...  If that was accidental, great, I trust it will be applied shortly.  If rejection _was_ the intent, I'd like the chance to defend my suggested changes before the conversation is marked "resolved".

Reply:
My apologies @TallTed, it was indeed an error on my part. All done now I hope.